tests/acceptance/config/behat.yml

```diff
@@ -253,6 +253,15 @@ default:
         - WebDavLockingContext:
         - PublicWebDavContext:
 
+    apiShare:
```

Comment:
Should we specify the suite as ng? so that we wont get confused with the old share tests
```suggestion
    apiSharingNg:
```